src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Startup.cs

```diff
@@ -104,6 +109,9 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+            // For use with publicly exposed servers that are not restricting the Host header.
```

Comment:
I'm actually not sure what this comment is trying to say.

Reply:
e.g. Kestrel on the edge.